bigbluebutton-html5/imports/startup/client/auth.js

```diff
@@ -61,7 +61,7 @@ export function joinRouteHandler(nextState, replace, callback) {
 
       replace({ pathname: path });
 
-      callback();
+      return callback();
```

Comment:
Is this the code change for "camera unshare bug"?

Reply:
yah